src/ProvidedTypes.fsi

```diff
@@ -267,11 +236,61 @@ namespace ProviderImplementation.ProvidedTypes
         /// Returns a type where the type has been annotated with the given types and/or units-of-measure.
         /// e.g. float<kg>, Vector<int, kg>
         static member AnnotateType: basic: Type * argument: Type list -> Type
+        
```

Comment:
Why was this moved down below its original declaration site?

Reply:
Because of the dependency chain

Reply:
@dsyme  
```
member PatchDeclaringType : x:ProvidedTypeDefinition -> unit
```
So ProvidedProperty needs to be declared alongside ProvidedTypeDefinition